config/wrapper.jinja2

```diff
@@ -0,0 +1,270 @@
+<!doctype html>
```

Comment:
I think this file would be better called `docs-template.jinja2` or something more descriptive.